rhessys/cycle/patch_daily_I.c

```diff
@@ -294,7 +294,7 @@ void		patch_daily_I(
 			patch[0].soil_defaults[0][0].porosity_0,
 			patch[0].soil_defaults[0][0].porosity_decay,
 			patch[0].sat_deficit_z,
-			patch[0].sat_deficit_z, 0.0) - patch[0].field_capacity;
+			patch[0].sat_deficit_z, 0.0) - patch[0].rootzone.field_capacity;
```

Comment:
@selimnairb Please let me know which version of 'develop' branch you are comparing with. I'm comparing the '83c646a' of 'xiaoli' branch with 'e302474' of 'develop' branch. On my computer, I can't find any difference between these branch. You may be comparing 'xiaoli' branch with a previous 'develop' branch. Let me know what version of 'develop' branch you are referring to. I'll checkout these file from your branch.